storage/libradb/src/state_store/mod.rs

```diff
@@ -118,7 +118,7 @@ impl StateStore {
         let mut iter = self
             .db
             .iter::<JellyfishMerkleNodeSchema>(Default::default())?;
-        iter.seek_to_first();
+        iter.seek_to_first().unwrap();
```

Comment:
It doesn't really matter in tests but can you use `?` here for consistency.

Reply:
done